frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -649,22 +689,34 @@ const MonitoringDashboardsPage: React.FC<MonitoringDashboardsPageProps> = ({ mat
         });
         dispatch(monitoringDashboardsPatchAllVariables(allVariables));
 
-        // Set time range options to their defaults since they may have been changed on the
-        // previous dashboard
-        dispatch(monitoringDashboardsSetEndTime(null));
-        dispatch(monitoringDashboardsSetTimespan(MONITORING_DASHBOARDS_DEFAULT_TIMESPAN));
+        // Set time range and poll interval options to their defaults or from the query params if available
+
+        refreshInterval &&
+          dispatch(monitoringDashboardsSetPollInterval(_.toNumber(refreshInterval)));
+        dispatch(monitoringDashboardsSetEndTime(_.toNumber(endTime) || null));
+        dispatch(
+          monitoringDashboardsSetTimespan(
+            _.toNumber(timeSpan) || MONITORING_DASHBOARDS_DEFAULT_TIMESPAN,
+          ),
+        );
```

Comment:
It looks like maybe we don't need these lines now? It was previously resetting these values to their defaults when you changed dashboard, but with this PR we actually want to persist the values, so no resetting should be required.